### DISCUSSION TITLE
Specific websites do not get grayscaled

### DISCUSSION THREAD

**Discussion by denizyavasxyz** [proginosko/LeechBlockNG#469] [Q&A] [answered]
Hi everyone!

I am trying to use two block sets at a time. One functions as a delay and the other should grayscale the page after the delay is done. 

Now the weird thing is that while using the Brave browser, Reddit gets delayed but not gray scaled after the delay. For all other sites including YouTube, Amazon, etc. it works. It just seems to be Reddit. I tried replicating the issue and configured the plugin using the Firefox browser with the exact same settings. On the Firefox browser, it works as expected. Reddit gets delayed first and gray scaled directly afterward.

Did anyone encounter a same issue and was able to solve it?

**Comment by proginosko** [comment]
Please export your options to a text file (Options > General > Export Options) and email it to me at leechblockng@proginosko.com so I can take a closer look. I'd like to test it with Chrome.

**Reply by denizyavasxyz** [reply]
Just sent it over.

**Reply by proginosko** [reply]
Sorry, I don't seemed to have received it. Checked my spam folder too. Can you double-check the email address? Thanks.

**Reply by denizyavasxyz** [reply]
Tried it with a different address.

**Comment by proginosko** [comment]
Thanks for sending the options file. I tested your setup with Chrome on Windows and it works as expected: Reddit is delayed and then grayscaled. So I'm guessing this is an issue with Brave specifically. I don't use Brave, but I'm willing to install it temporarily to try to figure out what might be causing the problem. Bear with me...

**Comment by proginosko** [comment, marked as answer]
Okay, I finally found time to look into this.

I installed Brave and was able to reproduce the problem using your exported options file. It turns out the delaying page has nothing to do with it. (You can disable that block set and you still don't get the grayscale with Reddit.) **Rather, it has to do with Brave's built-in tracker/ad blocking.** If you go to Reddit, click the Brave icon, and toggle the switch next to "Shields are UP" (i.e., change to "Shields are DOWN"), the grayscale will be applied correctly. If you reactivate "Shields are UP", open "Advanced controls", and change the option "Block trackers & ads" to "Allow all trackers & ads", the grayscale filter will work.

I don't know _why_ Brave's tracker/ad blocking causes this problem, but at least you have a workaround for now.
